### DISCUSSION TITLE
Validate how to?

### DISCUSSION THREAD

**Discussion by renrom** [Budibase/budibase#10067] [Help] [answered]
i have a form with some fields, there are 2 fiels in there that at least one of them have to be filled in.

Is it possible to validate the form that it checks that at least one of the two fields is filled?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @renrom,

I've looked into this, I was going to suggest using configure conditions. However, whenever trying to achieve what you requested I've encountered a bug. I've asked for a previous instance of this issue to be re-opened. Hopefully it will be fixed swiftly. The issue can be found here: https://github.com/Budibase/budibase/issues/5341

I hope this helps!

**Reply by ConorWebb96** [reply]
When this issue is fixed you will need to something similar to the following:

**Set up a validation rule that checks for a field being empty**
<img width="1338" alt="Screenshot 2023-03-21 at 08 52 42" src="https://user-images.githubusercontent.com/126772285/226557449-502a3253-dbf1-4925-808d-0e43889546d3.png">

**Validation rule which will be set whenever the condition is met**
<img width="1341" alt="Screenshot 2023-03-21 at 08 52 49" src="https://user-images.githubusercontent.com/126772285/226557455-dc5e2e02-b14c-4eca-ac1c-5971138730e6.png">

**Reference the field which has the value you are comparing against.**
<img width="1011" alt="Screenshot 2023-03-21 at 08 53 03" src="https://user-images.githubusercontent.com/126772285/226557459-53104975-f020-4739-8e71-7a1d2de6d858.png">

From here the required property should be added or removed depending on whether the field has text or not.

Hopefully this helps.
